### DISCUSSION TITLE
Where to download working arm64 lxc containers

### DISCUSSION THREAD

**Discussion by xifi-kif** [jiangcuo/Proxmox-Port#98] [Q&A]
I've a working Proxmox port on a orange pi running armbian, but I can't find a working container. I've tried bookworm downloaded here, but cointainers are not starting:
https://images.linuxcontainers.org/images/debian/bookworm/arm64/

While an old bullseye container I had is working, but I'd like a debian 12

Any help? :)

**Comment by caiotizio** [comment]
Hi,
you can find ct templates inside proxmox in local ->CT templates -> templates
to start a bookworm template you need to set network to static ip address , leaving  ip and gateway empty.
to set static ip edit `/etc/systemd/network/eth0.network` inside the container:

```
[Match]
Name=eth0
[Network]
DHCP=no
Address=192.168.1.104/24
Gateway=192.168.1.1
[DHCPv4]
UseDomains=true
```
set here IP and gateway as you need.

**Comment by spiritLHLS** [comment]
https://github.com/oneclickvirt/lxc_arm_images/releases/tag/debian

https://github.com/oneclickvirt/lxc_arm_images/releases/download/debian/debian_12_bookworm_arm64_cloud.tar.xz

this?
